extensions/blocks/testimonial/editor.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Internal dependencies
+ */
+import registerJetpackBlock from '../../utils/register-jetpack-block';
```

Comment:
Please rebase; as of https://github.com/Automattic/jetpack/pull/11971 this has been moved to `shared` from `utils`.